No module named six

When trying to use update_repo with `odklite`, I get a confusing error:

```
ODK_IMAGE=odklite TAG_IN_IMAGE=v1.3.2 ODK_DEBUG=yes ./run.sh make update_repo
Running  with -Xmx16G of memory for ROBOT and Java-based pipeline steps.
cpont.Makefile:36: warning: overriding recipe for target 'imports/mondo_import.owl'
Makefile:314: warning: ignoring old recipe for target 'imports/mondo_import.owl'
cpont.Makefile:49: warning: overriding recipe for target 'imports/uberon_import.owl'
Makefile:328: warning: ignoring old recipe for target 'imports/uberon_import.owl'
cpont.Makefile:65: warning: overriding recipe for target 'imports/ncit_import.owl'
Makefile:332: warning: ignoring old recipe for target 'imports/ncit_import.owl'
sh ../scripts/update_repo.sh
This (experimental) update script will create a new repo according to your config file. It will:
(1) overwrite your repositories Makefile, ODK sparql queries (your custom queries wont be touched) and docker wrapper (run.sh).
(2) and add missing files, if any.
Traceback (most recent call last):
  File "/tools/odk.py", line 15, in <module>
    from dataclasses_jsonschema import JsonSchemaMixin
  File "/usr/local/lib/python3.10/dist-packages/dataclasses_jsonschema/__init__.py", line 21, in <module>
    from .field_types import (  # noqa: F401
  File "/usr/local/lib/python3.10/dist-packages/dataclasses_jsonschema/field_types.py", line 8, in <module>
    from dateutil.parser import parse
  File "/usr/local/lib/python3.10/dist-packages/dateutil/parser/__init__.py", line 2, in <module>
    from ._parser import parse, parser, parserinfo, ParserError
  File "/usr/local/lib/python3.10/dist-packages/dateutil/parser/_parser.py", line 42, in <module>
    import six
ModuleNotFoundError: No module named 'six'
make: *** [Makefile:641: update_repo] Error 1
Command exited with non-zero status 2
### DEBUG STATS ###
Elapsed time: 0:00.38
Peak memory: 14532 kb
Please remember to update your ODK image from time to time: https://oboacademy.github.io/obook/howto/odk-update/.
```

Works fine:

```
ODK_IMAGE=odkfull TAG_IN_IMAGE=v1.3.2 ODK_DEBUG=yes ./run.sh make update_repo
```

Found this:
https://bobbyhadz.com/blog/python-no-module-named-six (not sure what to make of it)

For some reason adding six to the odklite requirements txt is not sufficient..

--- a/Makefile
+++ b/Makefile
@@ -35,6 +35,7 @@ test_odklite_programs:
 	@./tests/test-program.sh DOSDP-TOOLS dosdp-tools -v
 	@./tests/test-program.sh OWLTOOLS owltools --version
 	@./tests/test-program.sh AMMONITE sh amm --help
+	@./tests/test-program.sh ODK odk.py
 
 test_odkfull_programs: test_odklite_programs
 	@./tests/test-program.sh KONCLUDE Konclude -h

--- a/docker/odklite/Dockerfile
+++ b/docker/odklite/Dockerfile
@@ -25,6 +25,7 @@ RUN apt-get update && \
         openjdk-11-jre-headless \
         python3-pip \
         python-is-python3 \
+        python3-six \
         make \
         unzip \
         wget \

--- a/seed-via-docker.sh
+++ b/seed-via-docker.sh
@@ -3,5 +3,6 @@
 set -e
 
 ODK_IMAGE=${ODK_IMAGE:-odkfull}
+ODK_TAG=${ODK_TAG:-latest}
 
-docker run -v $HOME/.gitconfig:/root/.gitconfig -v $PWD:/work -w /work --rm obolibrary/$ODK_IMAGE /tools/odk.py seed "$@"
+docker run -v $HOME/.gitconfig:/root/.gitconfig -v $PWD:/work -w /work --rm obolibrary/$ODK_IMAGE:$ODK_TAG /tools/odk.py seed "$@"

--- a/tests/test-program.sh
+++ b/tests/test-program.sh
@@ -1,7 +1,8 @@
 #!/bin/bash
 
 ODK_IMAGE=${ODK_IMAGE:-odkfull}
+ODK_TAG=${ODK_TAG:-latest}
 
 program_name=$1; shift
 echo -n "Checking for $program_name... "
-docker run --rm -ti obolibrary/$IMAGE $@ >/dev/null && echo OK || echo KO
+docker run --rm -ti obolibrary/$ODK_IMAGE:$ODK_TAG $@ >/dev/null && echo OK || echo KO

--- a/tests/test-program.sh
+++ b/tests/test-program.sh
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-IMAGE=${IMAGE:-odkfull}
+ODK_IMAGE=${ODK_IMAGE:-odkfull}
 
 program_name=$1; shift
 echo -n "Checking for $program_name... "

